style.css

```diff
@@ -482,3 +484,121 @@ button {
   font-weight: bold;
   width: auto;
 }
+
+/* contact form */
+.contact-me {
+  width: 375px;
+  height: 790px;
+  background: #6070ff;
+  border-top-left-radius: 100px;
+  margin-top: -160px;
+  background-image: url(./images/contact-shapes.png);
+  background-repeat: no-repeat;
+  background-position-y: 9px;
+  background-position-x: 153px;
+}
+
+#contact-me {
+  width: auto;
+  height: 56px;
+  text-align: center;
+  margin-top: 78px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+#text {
+  width: 327px;
+  height: 110px;
+  text-align: center;
+
+  /* margin-top: 30px; */
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  margin: 24px auto;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 53px;
+}
+
+.fields {
+  display: flex;
+  flex-direction: column;
+
+  /* margin: 0 auto; */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain